cdap-ui/app/cdap/components/Replicator/Create/Content/Advanced/index.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import { createContextConnect, ICreateContext } from 'components/Replicator/Create';
+import StepButtons from 'components/Replicator/Create/Content/StepButtons';
+import WidgetWrapper from 'components/ConfigurationGroup/WidgetWrapper';
+import Heading, { HeadingTypes } from 'components/Heading';
+
+const styles = (): StyleRules => {
+  return {
+    root: {
+      padding: '30px 40px',
+    },
+    numInstances: {
+      width: '200px',
+    },
+  };
+};
+
+const OffsetBasePath = ({ onChange, value }) => {
```

Comment:
nit: It's a little confusing to me to have both OffsetBasePath and offsetBasePath - maybe this one and NumInstances could be named in a way to make it more clear that they are returning a widget/component?